src/modules/people-list/people-list-item.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import Swipeable from 'react-native-swipeable';
+import {IProps} from './people-list-item.type';
```

Comment:
same comment: make it `people-list-item.types` please. thanks

Reply:
I have changes people-list-item.types